### DISCUSSION TITLE
Backup/Export Scheme

### DISCUSSION THREAD

**Discussion by moriel5** [yogeshpaliyal/KeyPass#6] [Ideas]
Hello, I was quite intrigued when I saw your app on F-Droid, even though I personally do not make use of a password manager (I prefer to memorize them), and I saw that your KeyPass app supports backing up and restoring it's database.

Since I just saw a few similar apps, and either they did not feature backups, or they did, however they did not support an existing standard, and were no longer in development, so I was wondering whether you make use of Dominik Reichl's KeePass standard?

**Comment by yogeshpaliyal** [comment]
@moriel5 you want the KeyPass should support for backups from keepass and vice-versa

**Comment by moriel5** [comment]
Thanks for the answer, however I failed to understand it.

Do you mean that you do make use of it? Or that you thought that I had made a feature request? Or something else entirely?

**Comment by yogeshpaliyal** [comment]
@moriel5 no i an not using any reference or standards of KeePass

**Comment by moriel5** [comment]
Ah, thanks for the clarification.
In that case, it may be a good idea to add support for importing to/exporting from aforementioned standard, to make it easier for people to use the same database with your KeePass app at the same time as other apps, such as on the desktop.
